### PR TITLE
feat: add uzbekistan and districts

### PR DESCRIPTION
**Pull request by tiran23**
Added:
import uzbekistan from './countries/uzbekistan_regional.geojson'; import tashkent from './countries/tashkent_region_districts..geojson'; import tashkent_districts from './countries/tashkent_districts.geojson'; import surkhandarya_region_districts from './countries/surkhandarya_region_districts..geojson'; import sirdaryo_region_districts from './countries/sirdaryo_region_districts.geojson'; import jizzakh_region_districts from './countries/jizzakh_region_districts..geojson';

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts [hunk 1]
```diff
@@ -101,6 +101,12 @@ import usa from './countries/usa.geojson';
 import zambia from './countries/zambia.geojson';
 import venezuela from './countries/venezuela.geojson';
 import vietnam from './countries/vietnam.geojson';
+import uzbekistan from './countries/uzbekistan_regional.geojson';
+import tashkent from './countries/tashkent_region_districts..geojson';
```
Comment (villebro) [inline superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts:105]: Are the double periods here intentional?

File: superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts [hunk 2]
```diff
@@ -101,6 +101,12 @@ import usa from './countries/usa.geojson';
...
+import tashkent_districts from './countries/tashkent_districts.geojson';
+import surkhandarya_region_districts from './countries/surkhandarya_region_districts..geojson';
```
Comment (villebro) [inline superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts:107]: same here

File: superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts [hunk 3]
```diff
@@ -101,6 +101,12 @@ import usa from './countries/usa.geojson';
...
+import sirdaryo_region_districts from './countries/sirdaryo_region_districts.geojson';
+import jizzakh_region_districts from './countries/jizzakh_region_districts..geojson';
```
Comment (villebro) [inline superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts:109]: and here

**Review by villebro — COMMENTED** [review]
It appears your PR is missing the added `geojson` files.

**Comment by rusackas** [comment]
Closing this as it seems to have unresolved issues. Also, all new maps should be generated by the jupyter notebook, which is now much easier to do!

**Pull request closed without merging**